nautobot/extras/tables.py

```diff
@@ -69,7 +69,7 @@
 """
 
 JOB_BUTTONS = """
-<a href="{% url 'extras:job_run' slug=record.slug %}" class="btn btn-primary btn-xs" title="Run/Schedule" {% if not perms.extras.run_job or not record.enabled or not record.installed %}disabled="disabled"{% endif %}><i class="mdi mdi-play" aria-hidden="true"></i></a>
+<a href="{% url 'extras:job_run' slug=record.slug %}" class="btn btn-primary btn-xs" title="Run/Schedule" {% if not perms.extras.run_job or not record.enabled or record.sensitive and record.approval_required or not record.installed %}disabled="disabled"{% endif %}><i class="mdi mdi-play" aria-hidden="true"></i></a>
```

Comment:
```suggestion
<a href="{% url 'extras:job_run' slug=record.slug %}"
    class="btn btn-primary btn-xs" title="Run/Schedule"
    {% if not perms.extras.run_job or not record.enabled or record.sensitive and record.approval_required or not record.installed %}disabled="disabled"{% endif %}
><i class="mdi mdi-play" aria-hidden="true"></i></a>
```

Though honestly I'm not sure this check is needed - we should never get into this state, right?